### PR TITLE
add simple malloysql docs

### PR DESCRIPTION
**Pull request by bporterfield**

### CODE REVIEW

File: src/documentation/malloy_sql/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# MalloySQL
+
+MalloySQL enables mixing dialect-specific DDL with Malloy. For example, MalloySQL can be used to create a new table in BigQuery, based on a model of existing data:
```
Comment (carlineng) [inline src/documentation/malloy_sql/index.md:3]:
```suggestion
MalloySQL enables mixing dialect-specific SQL (including DDL) with Malloy. For example, MalloySQL can be used to create a new table in BigQuery, based on a model of existing data:
```

File: src/table_of_contents.json [hunk 2]
```diff
@@ -205,6 +205,15 @@
           "link": "/visualizations/segment_maps.md"
         }
       ]
+    },
+    {
+      "title": "MalloySQL",
+      "items": [
+        {
+          "title": "Using MalloyQL",
```
Comment (carlineng) [inline src/table_of_contents.json:213]:
```suggestion
          "title": "Using MalloySQL",
```

File: src/documentation/malloy_sql/index.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+```malloy
+>>>malloy
+import "raw_data.malloy"
+
+>>>sql connection:duckdb
+
+copy %{
+  raw_titles -> {
+    join_one: raw_ratings on tconst = raw_ratings.tconst
+    join_one: raw_crew on tconst = raw_crew.tconst
```
Comment (carlineng) [inline src/documentation/malloy_sql/index.md:79]: I think in the next iteration of the language syntax, `join_one` is not going to be allowed in a query block, and will need to be added in an `extend` block (see the [examples in this doc](https://docs.google.com/document/d/1hMT8WIHvFjcxCoAosrJHvixEj-yJ7Q3Is0CwRyXHPCw/edit)).

It *is* legal syntax today, and I think it won't break even when the new syntax is implemented (both old and new syntax will work, but the old will throw warnings), but I would consider getting rid of these joins so there's one fewer place we need to update the docs when the changes land.

**Review by carlineng — APPROVED** [review]
A couple of comments/suggestions, but overall LGTM!

File: src/documentation/malloy_sql/index.md (hunk 3 above)
Comment (bporterfield) [reply src/documentation/malloy_sql/index.md:79]: Don't have doc access, requested it. In the interest of getting this in before i'm OOO a bunch I'm gonna push this - don't trust myself to get it exactly right anyways (I stole this from Lloyd). Would love any other example you think might be useful!

**Pull request merged**